### PR TITLE
FunctionalMapoftheWorld

### PR DESCRIPTION
**Pull request by CharlesAuthier**

### CODE REVIEW

File: deep-learning/_posts/2018-08-13-FunctionalMapoftheWorld.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+author: Charles Authier
+pdf:  https://arxiv.org/pdf/1711.07846
+cite:
+  authors: "Gordon Christie, Neil Fendley, James Wilson, Ryan Mukherjee"
+  title:   "Functional Map of the World"
+  venue:   "CVPR 2018"
+---
+
+## Dataset
+Over 1 million images from over 200 countries. Each image, provide at least one bounding box annotation containing one of 63 categories.
```
Comment (SarahLECLERC7) [inline deep-learning/_posts/2018-08-13-FunctionalMapoftheWorld.md]: For each image, the goal is to provide? 
or 
There is at least one bounding box associated to one of the 63 possible categories for each image

File: deep-learning/_posts/2018-08-13-FunctionalMapoftheWorld.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+the dataset include:
```
Comment (SarahLECLERC7) [inline deep-learning/_posts/2018-08-13-FunctionalMapoftheWorld.md]: includes*

File: deep-learning/_posts/2018-08-13-FunctionalMapoftheWorld.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+- UTM Zone (zone of projection)
+- Timestamp (year, month, day, hour, minute, second, and day of the week UTC)
+- GSD (Ground sample distance)
+- Angles (the sensor is imaging the ground, as well as the angular location)
+- Image+box sizes
+
+3 types of images: 8/4/3 bands, 4 and 8 are RGB with multispectral images (MSI) all images are 256x256 with roughly 30cm resolution.
```
Comment (SarahLECLERC7) [inline deep-learning/_posts/2018-08-13-FunctionalMapoftheWorld.md]: 8 / 4 / 3 bands, 4 and 8 are RGB with multispectral images (MSI).
All images are 256x256 with a resolution of roughly 30 cm

File: deep-learning/_posts/2018-08-13-FunctionalMapoftheWorld.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+![](/deep-learning/images/FunctionalMapoftheWorld/class_world.png)
+
+## Meaning of the paper
+The paper present an analysis of the dataset along with a baseline approaches that reason about metadata and temporal views.
```
Comment (SarahLECLERC7) [inline deep-learning/_posts/2018-08-13-FunctionalMapoftheWorld.md]: presents*

Comment (SarahLECLERC7) [inline deep-learning/_posts/2018-08-13-FunctionalMapoftheWorld.md]: approach

Comment (SarahLECLERC7) [inline deep-learning/_posts/2018-08-13-FunctionalMapoftheWorld.md]: reasons*

Comment (SarahLECLERC7) [reply deep-learning/_posts/2018-08-13-FunctionalMapoftheWorld.md]: or baseline approaches

File: deep-learning/_posts/2018-08-13-FunctionalMapoftheWorld.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+Approaches:
+- LSTM-M An LSTM architecture trained using temporal sequences of metadata features.
+- CNN-I A standard CNN approach using only images, where DenseNet is fine-tuned after ImageNet. Softmax outputs are summed over each temporal view, after which an argmax is used to make the final prediction. The CNN is trained on all images across all temporal sequences of train + val.
```
Comment (SarahLECLERC7) [inline deep-learning/_posts/2018-08-13-FunctionalMapoftheWorld.md]: learning on ImageNet?

File: deep-learning/_posts/2018-08-13-FunctionalMapoftheWorld.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+- CNN-IM A similar approach to CNN-I, but with metadata features concatenated to the features of DenseNet before the fully connected layers.
+- LSTM-I An LSTM architecture trained using features extracted from CNN-I.
+- LSTM-IM An LSTM architecture trained using features extracted from CNN-IM.
+
+![](/deep-learning/images/FunctionalMapoftheWorld/images_world.png)
+
+## Results
```
Comment (SarahLECLERC7) [inline deep-learning/_posts/2018-08-13-FunctionalMapoftheWorld.md]: please say a word about the results

File: deep-learning/_posts/2018-08-13-FunctionalMapoftheWorld.md (hunk 1 above)
Comment (CharlesAuthier) [reply deep-learning/_posts/2018-08-13-FunctionalMapoftheWorld.md]: There is at least one bounding box associated to one of the 63 possible categories for each image.

File: deep-learning/_posts/2018-08-13-FunctionalMapoftheWorld.md (hunk 5 above)
Comment (CharlesAuthier) [reply deep-learning/_posts/2018-08-13-FunctionalMapoftheWorld.md]: ImageNet

File: deep-learning/_posts/2018-08-13-FunctionalMapoftheWorld.md (hunk 1 above)
Comment (SarahLECLERC7) [reply deep-learning/_posts/2018-08-13-FunctionalMapoftheWorld.md]: well you should change the sentence above :)

File: deep-learning/_posts/2018-08-13-FunctionalMapoftheWorld.md (hunk 6 above)
Comment (SarahLECLERC7) [reply deep-learning/_posts/2018-08-13-FunctionalMapoftheWorld.md]: I think the most impressive is that it does so for every structure? anyway

File: deep-learning/_posts/2018-08-13-FunctionalMapoftheWorld.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+![](/deep-learning/images/FunctionalMapoftheWorld/results_world.png)
+
+In conclusion, the concatenation of the metadata and the features of DenseNet help the classification of world images.
```
Comment (lemairecarl) [inline deep-learning/_posts/2018-08-13-FunctionalMapoftheWorld.md]: Overall, it seems that the LSTM helps, but just a bit. Maybe the advantage is clearer when looking at corner cases, or by looking a the precision/recall?

**Pull request merged**